client/views/user_name.tpl

```diff
@@ -0,0 +1,5 @@
+{{#if from}}
```

Comment:
Since you created a username template, you could pass in the string into it, then you won't need three different span generations.